astropy/io/fits/scripts/fitscheck.py

```diff
@@ -46,15 +46,19 @@
 import warnings
 
 from astropy.io import fits
+from astropy import version
 
 log = logging.getLogger('fitscheck')
 
-_DESCRIPTION = """
+_DESCRIPTION = f"""
 e.g. fitscheck example.fits
 
 Verifies and optionally re-writes the CHECKSUM and DATASUM keywords
 for a .fits file.
 Optionally detects and fixes FITS standard compliance problems.
+
+This script is part of the Astropy package, version {version.major}.{version.minor}.{version.bugfix}. See
```

Comment:
Might be simpler if you use `from astropy import __version__`? Not sure if a `try ... except ...` for `ImportError` is necessary or not.

Reply:
Now using `from astropy import __version__`, which indeed is simpler.